sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/LocalDocumentStorage.java

```diff
@@ -221,34 +223,36 @@ private static SQLiteQueryBuilder getPartitionAndDocumentIdQueryBuilder() {
     }
 
     /**
-     * Creates or overwrites specified document entry in the cache. Sets the de-serialized value of
-     * the document to null and the pending operation to DELETE.
+     * Add delete pending operation to a document.
      *
-     * @param partition  Partition key.
-     * @param documentId Document id.
-     * @return True if cache was successfully written to, false otherwise.
+     * @param table      table.
+     * @param partition  partition.
+     * @param documentId document identifier.
+     * @return true if storage update succeeded, false otherwise.
      */
-    boolean markForDeletion(String table, String partition, String documentId) {
+    boolean deleteOffline(String table, String partition, String documentId) {
```

Comment:
Sounds the method is deleting offline. We need better name for the method.

Reply:
We need better methods for the entire class, I just kept consistency for now until we decide better names.